phpmd.xml

```diff
@@ -18,10 +18,10 @@
 
     <rule ref="rulesets/naming.xml/ConstantNamingConventions" />
 
-    <rule ref="vendor/mi-schi/phpmd-extension/rulesets/cleancode.xml/DataStructureMethods" />
-    <rule ref="vendor/mi-schi/phpmd-extension/rulesets/cleancode.xml/SwitchStatement" />
+    <rule ref="../../../../../mi-schi/phpmd-extension/rulesets/cleancode.xml/DataStructureMethods" />
```

Comment:
To me, `.dist` means "sane defaults". That would make explicit that the file may not match all cases and that one may use different settings. Anyway, if paths are relative to phpmd installtion, then it should always work.